implementations/sbcl-setup.html

```diff
@@ -34,7 +34,10 @@ <h2 id="linedit">Linedit</h2>
     (require :linedit)
     (funcall (intern &quot;INSTALL-REPL&quot; :linedit) :wrap-current t))</code></pre>
 <h2 id="installing-on-windows">Installing on Windows</h2>
-<p>Dear Windows SBCL user, please tell us how it&#8217;s done!</p>
+<p> Install the Scoop package manager. To do this, open PowerShell and type: ' iex (new-object net.webclient).downloadstring('https://get.scoop.sh') ' (minus single quotes).
+Grant whatever permissions it needs. </p>
```

Comment:
this is not a good phrase, can I ask for a rewording along the lines of:

```suggestion
"please review the permissions for scoop and and accept if you are comfortable with them</p>
```